frontend/src/pages/answer/fullKg/simulation.worker.js

```diff
@@ -0,0 +1,21 @@
+/** Web Worker to run d3 force simulation and return placed nodes and edges */
+import * as d3 from 'd3';
+
+onmessage = (e) => {
+  const { edges, nodes } = e.data;
+  const simulation = d3.forceSimulation(nodes)
+    .force('charge', d3.forceManyBody())
+    .force('collide', d3.forceCollide(10).strength(0.5))
+    .force('link', d3.forceLink(edges).id((d) => d.id).distance(50).strength(1))
+    .force('forceX', d3.forceX())
+    .force('forceY', d3.forceY())
```

Comment:
Could we document what each of these forces is meant to do?

Reply:
Added in b45c753b1a59f99d276aca61585e4c6c2fca9bb9

Reply:
"create global electrostatic charge" doesn't really tell me what's going to happen. Is this attracting the nodes to each other? I'm also not sure what "add edges with length" means? Is it displaying the KG edges here?

Reply:
Updated in f6124cbdc4c10c69c3af9d968c9e141558158dcc.